controller/RestController.php

```diff
@@ -200,6 +200,7 @@ public function vocabularyInformation($request)
                 'skos' => 'http://www.w3.org/2004/02/skos/core#',
                 'onki' => 'http://schema.onki.fi/onki#',
                 'dct' => 'http://purl.org/dc/terms/',
+                'dcterms' =>'http://purl.org/dc/terms/',
```

Comment:
Why do you need to define this prefix? `dct` is defined on the line above, you can just use that.